diff --git a/node_modules/@ngx-formly/core/esm2020/lib/core.module.mjs b/node_modules/@ngx-formly/core/esm2020/lib/core.module.mjs
index 526037f..0eab36c 100644
--- a/node_modules/@ngx-formly/core/esm2020/lib/core.module.mjs
+++ b/node_modules/@ngx-formly/core/esm2020/lib/core.module.mjs
@@ -18,7 +18,6 @@ import * as i1 from "./services/formly.config";
 export function defaultFormlyConfig(config) {
     return {
         types: [
-            { name: 'formly-group', component: FormlyGroup },
             { name: 'formly-template', component: FormlyTemplateType },
         ],
         extensions: [
diff --git a/node_modules/@ngx-formly/core/fesm2015/ngx-formly-core.mjs b/node_modules/@ngx-formly/core/fesm2015/ngx-formly-core.mjs
index 812e29c..67ddc7a 100644
--- a/node_modules/@ngx-formly/core/fesm2015/ngx-formly-core.mjs
+++ b/node_modules/@ngx-formly/core/fesm2015/ngx-formly-core.mjs
@@ -2284,7 +2284,6 @@ class CoreExtension {
 function defaultFormlyConfig(config) {
     return {
         types: [
-            { name: 'formly-group', component: FormlyGroup },
             { name: 'formly-template', component: FormlyTemplateType },
         ],
         extensions: [
diff --git a/node_modules/@ngx-formly/core/fesm2020/ngx-formly-core.mjs b/node_modules/@ngx-formly/core/fesm2020/ngx-formly-core.mjs
index 682972b..e745c79 100644
--- a/node_modules/@ngx-formly/core/fesm2020/ngx-formly-core.mjs
+++ b/node_modules/@ngx-formly/core/fesm2020/ngx-formly-core.mjs
@@ -2245,7 +2245,6 @@ class CoreExtension {
 function defaultFormlyConfig(config) {
     return {
         types: [
-            { name: 'formly-group', component: FormlyGroup },
             { name: 'formly-template', component: FormlyTemplateType },
         ],
         extensions: [
